src/game.js

```diff
@@ -0,0 +1,43 @@
+var Game = function(
+  frameOne = new BowlingFrame(),
+  scoreboard = new Scoreboard()
+) {
+  this.frameOne = frameOne;
+  this.scoreboard = scoreboard;
+  this.currentFrame = this.frameOne;
+};
+
+Game.prototype = {
+  readScore: function(frame) {
+    return frame.total();
+  },
+
+  bowl: function(frame) {
+    if (typeof frame.firstRollScore == 'number') {
+      return frame.secondRoll();
+    }
+    var score = frame.firstRoll();
+    this.strikeTracker(frame);
+    return score;
+  },
+
+  strikeTracker: function(frame) {
+    if (frame.firstRollScore === 10) {
+      frame.secondRollScore = 0;
+      return true;
+    }
+    return false;
+  },
+
+  spareTracker: function(frame) {
+    if (frame.total() === 10) {
+      return true;
+    }
+    return false;
+  },
+
+  createNewFrame: function() {
+    this.frameTwo = new BowlingFrame();
```

Comment:
This feels a little confusing to me - after the first frame is completed, it seems like every subsequent frame is going to be stored in the frameTwo instance variable which doesn't feel right.  I think you could probably do away with the frameOne and frameTwo instance variable altogether and just rely on the currentFrame to track where you are in the game.